tests/dualtor/test_tunnel_memory_leak.py

```diff
@@ -66,30 +66,29 @@ def is_tunnel_packet_handler_running(duthost):
     return status == 'RUNNING'
 
 
-def get_memory_info(duthost):
-    stdout_lines = duthost.command("docker stats swss --no-stream")["stdout_lines"]
-    header = stdout_lines[0]
-    # Find the position of category "MEM USAGE", "MEM %" and "NET I/O"
-    pos1 = header.index("MEM USAGE")
-    pos2 = header.index("MEM %")
-    pos3 = header.index("NET I/O")
-    if len(stdout_lines) < 2:
-        pytest.fail("Collect swss stat failed, swss container may die.")
-    line = stdout_lines[-1]
-    # Get the value of "MEM USAGE", "LIMIT" and "MEM %"
-    mem_info = line[pos1:pos2].strip().split("/")
-    mem_usage = mem_info[0].strip()
-    mem_limit = mem_info[1].strip()
-    mem_percent = line[pos2:pos3].strip()
-    return mem_usage, mem_limit, mem_percent
-
-
-def check_memory_leak(duthost, target_mem_percent, delay=10, timeout=15, interval=5):
+def get_tunnel_packet_handler_memory_usage(duthost):
+    pid_command = "ps -ef | grep tunnel_packet_handler.py | grep -v grep | awk '{print $2}'"
+    pid_output = duthost.shell(pid_command)["stdout"]
+    if not pid_output:
```

Comment:
Suggest using command `pidof` to get the pid.

Reply:
@bingwang-ms  can not get pid through pidof for tunnel_packet_handler.py

```
admin@str2-7260cx3-acs-10:~$ pidof tunnel_packet_handler.py
admin@str2-7260cx3-acs-10:~$ 

admin@str2-7260cx3-acs-10:~$ ps -axu | grep tunnel
root      897133  0.0  0.0  88284  7840 pts/0    Sl   Oct23   0:03 /usr/bin/tunnelmgrd
root     1065269  0.0  0.8 162192 70412 pts/0    Sl   Oct23   0:04 python3 /usr/bin/tunnel_packet_handler.py
admin    2018389  0.0  0.0   6972  2180 pts/0    S+   01:30   0:00 grep tunnel
admin@str2-7260cx3-acs-10:~$ pidof /usr/bin/tunnel_packet_handler.py
admin@str2-7260cx3-acs-10:~$ 

```